sway-core/src/semantic_analysis/ast_node/declaration/function.rs

```diff
@@ -226,7 +226,6 @@ impl TypedFunctionDeclaration {
             is_contract_call: ctx.mode() == Mode::ImplAbiFn,
             purity,
         };
-
```

Comment:
Whoops, did I do this with my suggestion?  It's unrelated to your PR so we should probably leave this file alone and keep the empty line.

Reply:
I might have touched it accidentally, will fix it in a minute